bundle/pom.xml

```diff
@@ -1,4 +1,24 @@
 <?xml version="1.0" encoding="UTF-8"?>
+<!--

```

Comment:
could you refer to any spec or doc about putting the license into the pom.xml? I'm not sure if that's a right approach and would prefer a confirmation